literature.bib

```diff
@@ -18086,6 +18176,18 @@ @incollection{R03
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2002 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@article{AVM:SAGE02,
+  title={{Bravais-Pearson and Spearman Correlation Coefficients: Meaning, Test of Hypothesis and Confidence Interval}},
+  author={Artusi, R and Verderio, P and Marubini, EJTIjobm},
+  journal={The International journal of biological markers},
+  volume={17},
+  number={2},
+  pages={148--151},
+  year=2002,
+  publisher={SAGE Publications Sage UK: London, England}
+}
```

Comment:
Githubs syntax highlighting stopped working here. Might there be a syntax somewhere above this entry? Also inconsistent style